server/index.js

```diff
@@ -35,6 +36,10 @@ app.use(RequestSyntaxErrorHandler);
 
 startServer();
 
+// ----- Start cron tasks
+removeDuels.start();
+checkDuels.start();
```

Comment:
I suggest to rename these
```suggestion
removeDuelsTask.start();
checkDuelsTask.start();
```